### PR TITLE
Feature/add toaster notification

### PR DESCRIPTION
**Pull request by ivanzusko**

### CODE REVIEW

File: src/modules/modals/Toaster/Toaster.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        props.clearConfirmationModal();
+    };
+
+    const closeLater = () => {
+        setTimeout(() => {
+            handleCloseToaster();
+        }, props.timeout || 2000);
+    };
+
+    if (props.isOpen) {
```
Comment (rafaeljesus) [inline src/modules/modals/Toaster/Toaster.js]: here you could just
```js
if (!props.isOpen) return;

closeLater();

return (
  <div className={b()}>
  ....
)

```

File: src/modules/modals/Toaster/Toaster.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                    />
+                </div>
+            </div>
+        );
+    }
+
+    return null;
+};
+
+const mapStateToProps = (state) => {
```
Comment (rafaeljesus) [inline src/modules/modals/Toaster/Toaster.js]: perhaps `state => `?

File: src/modules/modals/Toaster/Toaster.js (hunk 1 above)
Comment (ivanzusko) [reply src/modules/modals/Toaster/Toaster.js]: Dammit :) Totally agree 👍 👍 👌

Comment (ivanzusko) [reply src/modules/modals/Toaster/Toaster.js]: Dammit :) Totally agree 👍 👍 👍 👌

**Pull request merged**